tests/basic/determinism_check.rs

```diff
@@ -0,0 +1,98 @@
+// use crate::check_replay_roundtrip;
+use shuttle::rand::{thread_rng, Rng};
+// use shuttle::scheduler::RandomScheduler;
+use shuttle::sync::{Arc, Mutex};
+use shuttle::{check_determinism, thread};
+use test_log::test;
+
+#[test]
+#[should_panic]
+fn shuttle_determinism_test_randomly_acquire_lock() {
```

Comment:
The tests in this file don't need to start with `shuttle_determinism_test` (that's implied by the module we're in).